docs/StardustDocs/topics/DataColumn.md

```diff
@@ -1,12 +1,14 @@
 [//]: # (title: DataColumn)
 <!---IMPORT org.jetbrains.kotlinx.dataframe.samples.api.Create-->
 
-[`DataColumn`](DataColumn.md) represents a column of values. It can store objects of primitive or reference types, or other [`DataFrames`](DataFrame.md).
+[`DataColumn`](DataColumn.md) represents a column of values.
+It can store objects of primitive or reference types, 
+or other [`DataFrame`](DataFrame.md) objects.
 
 See [how to create columns](createColumn.md)
 
 ### Properties
-* `name: String` — name of the column, should be unique within containing dataframe
+* `name: String` — name of the column should be unique within containing dataframe
```

Comment:
maybe `;` is needed here? It's two sentences:
name of the column
should be unique within containing dataframe

Reply:
yes, ; or two sentences